### PR TITLE
Home Articles

### PR DESCRIPTION
**Pull request by AKidd95**
Pull Request template. 

#### Description
Changes proposed in this pull request:

* display article title on home page
* link article title to respective show page
* add content snippet
#### What I have learned working on this feature: 
* Strengthen proper string interpolation in haml
* explored iteration with EachDo and truncate

#### Screenshots:
screen shot 2018-03-18 at 3 32 46 pm

### CODE REVIEW

**Review by davidboll — APPROVED** [review]
Good job!

**Review by isabellelii — APPROVED** [review]

**Review by kmthorsnes — APPROVED** [review]
Looks good to me

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 class HomeController < ApplicationController
   def index
+    @article = Article.all
```
Comment (magnus-thor) [inline app/controllers/home_controller.rb]: This variable keeps all of the articles and thus should be plural

**Review by magnus-thor — APPROVED** [review]
Looks good

**Pull request merged**